public/js/components/RightSidebar.js

```diff
@@ -144,24 +144,24 @@ const RightSidebar = React.createClass({
     const { shouldPauseOnExceptions, shouldIgnoreCaughtExceptions,
             pauseOnExceptions } = this.props;
 
-    function pauseBtn(pause, ignore, tooltip) {
+    function pauseBtn(pause, ignore, tooltip, className) {
       return debugBtn(
         () => pauseOnExceptions(pause, ignore),
-        "pause-exceptions",
+        className,
         "enabled",
         tooltip
       );
     }
 
     if (!shouldPauseOnExceptions && !shouldIgnoreCaughtExceptions) {
```

Comment:
I think, using "className" as a parameter name was wrong decision and also consufing. In fact, it should be named "type". I guess removing pauseBtn function and using directly DebugBtn function much proper way as you said. In this way, we dont need any css class. Svg files changes according to the type parameter.  

https://github.com/devtools-html/debugger.html/blob/master/public/js/components/RightSidebar.js#L26